### DISCUSSION TITLE
Projects (beta): Control the "tab" view order

### DISCUSSION THREAD

**Discussion by abernix** [community/community#4803] [Projects and Issues] [answered]
The tabs that appear at the top of the new Projects (beta) board which represent different views seem to be positioned in the order that they're created.

I find myself wanting to re-order them after I've created them.  I see both semantic ordering and "default tab" as plausible options.  However, since "Default tab" could be inferred by having it be the "first" tab, I think I'm just hoping for the ability to reorder them, similar to how "fields" can have their options sorted.

**Comment by kevinwhoffman** [comment]
## Current Behavior

I like to group together views such as `Status` and `Target` which give me a high-level look at the project along with more specific views for each developer. In my current project, I realized that I forgot to create a `Target` view, and now I have no means to position that tab where I want it.

![CleanShot 2021-09-04 at 16 49 35](https://user-images.githubusercontent.com/1938671/132107755-dce3ef03-a634-41a3-a716-059ec3be341e.png)

## Desired Behavior

Here is how Airtable handles reordering and renaming:

- Reordering - click and hold to drag
- Renaming - double click or select "Rename" from dropdown menu

![CleanShot 2021-09-04 at 17 06 15](https://user-images.githubusercontent.com/1938671/132108048-2c026638-e202-4b64-a3f5-19bc6d01f078.gif)

**Reply by mitchelsellers** [reply]
This is a great idea, and would solve my issue 100%

**Reply by imlazyeye** [reply]
+1 -- greatly needed on my project

**Reply by devlachapelle** [reply]
+1 yes please!

**Reply by nicholasconfer** [reply]
+1 very important feature

**Reply by zolter** [reply]
+1

**Reply by matthewmayer** [reply]
+1

**Reply by MichaelKaramanolis** [reply]
agreed

**Reply by badrigit** [reply]
+1

**Comment by alexcnichols** [comment]
👋🏼 Improvements for re-ordering views, and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/283 :octocat:

**Comment by azenMatt** [comment, marked as answer]
Hi everyone - we just released re-orderable view tabs as part of our [recent updates](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/). 

You can now drag and drop the views into any order you want 😄

**Reply by mitchelsellers** [reply]
This is awesome, thanks for the quick turnaround on these types of feature requests.

**Reply by stoutgz** [reply]
@azenMatt this feature is broken in the latest for me. I don't seem to be able to find a drag cursor and I have tried clicking and dragging in just about every pixel I could.

- OS: macOS 13.0.1
- Browsers:
    - Brave Browser [Version 1.46.134 Chromium: 108.0.5359.94 (Official Build) (arm64)](https://brave.com/latest/)
    - Safari Version 16.1 (18614.2.9.1.12)
    

https://user-images.githubusercontent.com/3385153/207474084-959a1764-02fc-4fcd-b6e2-7d0b0b739864.mov
